common/src/main/java/org/apache/cassandra/sidecar/common/CassandraAdapterDelegate.java

```diff
@@ -174,6 +176,13 @@ public List<NodeStatus> getStatus()
         return adapter.getStatus();
     }
 
+    @Override
+    public StorageOperations storageOperations()
+    {
+        checkSession();
```

Comment:
nit: remove the check. JMX operation does not require session to be established, although there is no harm to check here.